iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/HelmProcessor.java

```diff
@@ -86,6 +87,12 @@ String processHelmTemplate(String path, String source, HelmInputFileContext inpu
   }
 
   static Map<String, String> validateAndReadFiles(HelmInputFileContext inputFileContext) {
+    var lineSeparators = List.of("\n", "\r\n");
```

Comment:
I will add here also `\r`, \u2028, and \u2029 to be on safe side. Some filesystems potentially allows such names: https://en.wikipedia.org/wiki/Comparison_of_file_systems#Limits